brushfire-core/src/main/scala/com/stripe/brushfire/Tree.scala

```diff
@@ -1,6 +1,7 @@
 package com.stripe.brushfire
 
 import com.twitter.algebird._
+import scala.collection.mutable
```

Comment:
without having actually seen what you're using this for: in general using mutable collections is a red flag for me in scala